docs/reference/ilm/apis/slm-api.asciidoc

```diff
@@ -591,3 +590,152 @@ This API will immediately return, as retention will be run asynchronously in the
 }
 --------------------------------------------------
 
+[[slm-stop]]
+=== Stop Snapshot Lifecycle Management API
+
+[subs="attributes"]
+++++
+<titleabbrev>Stop Snapshot Lifecycle Management</titleabbrev>
+++++
+
+Stop the Snapshot Lifecycle Management (SLM) plugin.
+
+==== Request
+
+`POST /_ilm/stop`
+
+==== Description
+
+Halts all snapshot lifecycle management operations and stops the SLM plugin.
+This is useful when you are performing maintenance on the cluster and need to
+prevent SLM from performing any actions on your indices. Note that this API does
+not stop any snapshots that are currently in progress.
```

Comment:
We should add a note that manually executing a policy or retention still works even with SLM stopped